src/app/modules/side-menu/components/filter-menu/filter-menu.component.html

```diff
@@ -64,55 +64,8 @@
           </div>
         </ion-accordion>
 
-        <!-- Snøskredregioner -->
-        <div *ngIf="isSupported('region') && isGeohazardSnow()">
-          <div *ngIf="regions$ | async as regions" slot="content">
-            <ion-accordion value="a-regions">
-              <ion-item slot="header" color="light">
-                <app-selected-items-counter-label [selectedItemsCount]="aRegionsSelectedCount()">
-                  {{ "OBSERVATION_FILTER.REGION_FILTER_TITLE_SNOW" | translate }}
-                </app-selected-items-counter-label>
-              </ion-item>
-              <div slot="content">
-                <ion-list lines="none">
-                  <ion-item *ngFor="let region of regions.a; trackBy: avalancheRegionTrackById">
-                    <ion-checkbox
-                      [value]="region"
-                      [checked]="region.checked"
-                      (ionChange)="regionCheckBoxChanged($event)"
-                      labelPlacement="end"
-                      justify="start"
-                    >
-                      {{ region.name }}
-                    </ion-checkbox>
-                  </ion-item>
-                </ion-list>
-              </div>
-            </ion-accordion>
-            <ion-accordion value="b-regions">
-              <ion-item slot="header" color="light">
-                <app-selected-items-counter-label [selectedItemsCount]="bRegionsSelectedCount()">
-                  {{ "OBSERVATION_FILTER.REGION_FILTER_B_REGIONS_TITLE" | translate }}
-                </app-selected-items-counter-label>
-              </ion-item>
-              <div slot="content">
-                <ion-list lines="none">
-                  <ion-item *ngFor="let region of regions.b; trackBy: avalancheRegionTrackById">
-                    <ion-checkbox
-                      [value]="region"
-                      [checked]="region.checked"
-                      (ionChange)="regionCheckBoxChanged($event)"
-                      labelPlacement="end"
-                      justify="start"
-                    >
-                      {{ region.name }}
-                    </ion-checkbox>
-                  </ion-item>
-                </ion-list>
-              </div>
-            </ion-accordion>
-          </div>
-        </div>
+        <app-region-filter *ngIf="isSupported('region')"></app-region-filter>
```

Comment:
Skal vi bruke den nye syntaksen? Siden den ikke brukes ellers i templaten så kan vi beholde det som det er også.

```suggestion
        @if (isSupported('region') {
            <app-region-filter *ngIf="isSupported('region')"></app-region-filter>
        }
```